### PR TITLE
Test /api/admin/sslCerts

### PR DESCRIPTION
**Pull request by Wolfy64**

### CODE REVIEW

File: src/tests/integration/admin.test.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    })
+
+    const request = await sslCertsMock()
+    expect(sslCertsMock).toHaveBeenCalledTimes(1)
+    expect(request).toEqual({
+      success: true,
+      message: 'SSL Certs and domain name records successfully created'
+    })
+  })
+})
```
Comment (Wolfy64) [inline src/tests/integration/admin.test.ts]: I'm not really sure how to test this api...

File: src/tests/api.test.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { startAppServer } from '../server/server'
 import fetch from 'node-fetch'
 
-const TEST_PORT = process.env.PORT || 4998
+const TEST_PORT = 4242
```
Comment (songz) [inline src/tests/api.test.ts]: sometimes, when I run tests, I run it on a different port. I have apps running on 4242 on my local machine, so `process.env.PORT` must remain.

File: src/tests/integration/admin.test.ts (hunk 1 above)
Comment (songz) [reply src/tests/integration/admin.test.ts]: haha you are testing to see if `mockResorvedValue` works. I don't think there's a way to test `/sslCerts` path.

File: src/tests/api.test.ts (hunk 2 above)
Comment (Wolfy64) [reply src/tests/api.test.ts]: I got it. I removed it because it circleCI was failing... 🧐

Comment (songz) [reply src/tests/api.test.ts]: hmmmm.... that sucks.
maybe `process.env.PORT || 4242`?

Comment (songz) [reply src/tests/api.test.ts]: or `process.env.TEST_PORT || 4242`

**Pull request closed without merging**